<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QTestMacroToolDlg</class>
 <widget class="QDialog" name="QTestMacroToolDlg">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>915</width>
    <height>668</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>选择测试功能</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="m_layoutTest" stretch="3,1">
     <item>
      <layout class="QVBoxLayout" name="m_layoutLeft">
       <item>
        <widget class="QScrollArea" name="scrollArea">
         <property name="widgetResizable">
          <bool>true</bool>
         </property>
         <widget class="QWidget" name="m_tableTest">
          <property name="geometry">
           <rect>
            <x>0</x>
            <y>0</y>
            <width>659</width>
            <height>603</height>
           </rect>
          </property>
          <property name="styleSheet">
           <string notr="true"/>
          </property>
          <layout class="QGridLayout" name="gridLayout"/>
         </widget>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="m_layoutRight">
       <item>
        <widget class="QScrollArea" name="scrollArea_2">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="widgetResizable">
          <bool>true</bool>
         </property>
         <widget class="QWidget" name="scrollAreaWidget">
          <property name="geometry">
           <rect>
            <x>0</x>
            <y>0</y>
            <width>217</width>
            <height>603</height>
           </rect>
          </property>
          <layout class="QGridLayout" name="gridLayout_2">
           <property name="margin">
            <number>0</number>
           </property>
           <item row="0" column="1">
            <widget class="QTreeWidget" name="m_treeTest">
             <property name="selectionMode">
              <enum>QAbstractItemView::NoSelection</enum>
             </property>
             <property name="selectionBehavior">
              <enum>QAbstractItemView::SelectItems</enum>
             </property>
             <property name="columnCount">
              <number>1</number>
             </property>
             <attribute name="headerVisible">
              <bool>false</bool>
             </attribute>
             <column>
              <property name="text">
               <string/>
              </property>
             </column>
            </widget>
           </item>
          </layout>
         </widget>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="m_layoutAdd">
     <item>
      <widget class="QPushButton" name="m_btnAddChild">
       <property name="text">
        <string>添加至:子项</string>
       </property>
       <property name="autoDefault">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="m_btnAddBefore">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>添加至:项之前</string>
       </property>
       <property name="autoDefault">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="m_btnAddAfter">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>添加至:项之后</string>
       </property>
       <property name="autoDefault">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="m_txtTimes">
       <property name="maximumSize">
        <size>
         <width>70</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="text">
        <string>测试次数：</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="m_editTimes">
       <property name="maximumSize">
        <size>
         <width>70</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="text">
        <string>1</string>
       </property>
       <property name="echoMode">
        <enum>QLineEdit::Normal</enum>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="m_editName">
       <property name="maximumSize">
        <size>
         <width>150</width>
         <height>16777215</height>
        </size>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="m_chkRptTitle">
       <property name="text">
        <string>作为Word标题</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="m_txtTitleLevel">
       <property name="text">
        <string>标题等级</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="m_cmbTitleLevel"/>
     </item>
     <item>
      <widget class="QPushButton" name="m_btnClose">
       <property name="maximumSize">
        <size>
         <width>100</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="text">
        <string>关闭</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
